lib/flipper/railtie.rb

```diff
@@ -10,6 +10,12 @@ class Railtie < Rails::Railtie
       )
     end
 
+    initializer "flipper.identifier" do
+      ActiveSupport.on_load(:active_record) do
+        ActiveRecord::Base.include Flipper::Identifier
+      end
+    end
+
     initializer "flipper.default", before: :load_config_initializers do |app|
```

Comment:
something seems wrong with how we're setting up this test or class.  I was poking around with `binding.pry` and noticed that `self` is `nil` within this initializer.  it should be the Railtie, which would allow us to do `config.before_initialize do ...` here, which seems more in line with how Rails initializes, eg. https://github.com/rails/rails/blob/main/activesupport/lib/active_support/railtie.rb#L20

Reply:
@dpep good find! Let's look into that in another PR. This one has set for too long already, so I'm going to try to get it merged soon.